src/test/java/org/isf/radiology/rest/RadiologyControllerTest.java

```diff
@@ -0,0 +1,210 @@
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.isf.radiology.rest;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.log;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import org.isf.OpenHospitalApiApplication;
+import org.isf.orthanc.model.Instance;
+import org.isf.orthanc.model.InstanceResponse;
+import org.isf.orthanc.model.Patient;
+import org.isf.orthanc.model.Series;
+import org.isf.orthanc.model.SeriesResponse;
+import org.isf.orthanc.model.Study;
+import org.isf.orthanc.model.StudyResponse;
+import org.isf.orthanc.service.OrthancAPIClientService;
+import org.isf.patient.data.PatientHelper;
+import org.isf.patient.manager.PatientBrowserManager;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.web.servlet.MockMvc;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Radiology Controller Test
+ * @author Silevester D.
+ */
+@SpringBootTest(classes = OpenHospitalApiApplication.class)
+@AutoConfigureMockMvc
+public class RadiologyControllerTest {
+	private final Logger LOGGER = LoggerFactory.getLogger(RadiologyControllerTest.class);
+
+	@Autowired
+	private MockMvc mvc;
+
+	@MockBean
+	private OrthancAPIClientService orthancAPIClientService;
+
+	@MockBean
+	private PatientBrowserManager patientBrowserManager;
+
+	@Autowired
+	private ObjectMapper objectMapper;
+
+	@Test
+	@WithMockUser(username = "admin", authorities = {"radiology.read"})
+	@DisplayName("Should successfully get a patient studies")
+	void testGetPatientStudiesById() throws Exception {
+		List<StudyResponse> studies = generateStudies();
+
+		when(patientBrowserManager.getPatientById(anyInt())).thenReturn(PatientHelper.setup());
+		when(orthancAPIClientService.getPatientStudiesById(anyString())).thenReturn(studies);
+
+		var result = mvc.perform(get("/radiology/patients/{id}/studies", "123")
```

Comment:
while `var` is valid, the project standards require strongly typed objects.   There are no other occurrences currently in the project and there seems no value to starting it now.  Thanks.